src/codenamet.scp

```diff
@@ -0,0 +1,54 @@
+func CODE_NAME_T_AUTO_ATTACK(from, self, adddam, arg)
+{
+    if (self.ClassName == 'PC'  from.ClassName != 'PC')
+    {
+        return false, 0, 0, 0, 0;
+    }
+
+    if (from.Job != 288)
+    {
+        return false, 0, 0, 0, 0;
+    }
+
+    // Get extra str and dex for attacker and mix them by using exponent
+    // As CNT is a ranged character that use both str and dex. It is more reasonable to add up both value then using the higher one.
+    var str = GET_EXTRA_STR(from);
+    var dex = GET_EXTRA_DEX(from);
+    
+    var mix = Math.floor(((str^2) + (dex^2))^0.5);
+
+    // Scaling bonus for 2% of extra mixed value
+    var bonusDamage = mix * 0.02;
+
+    return true, bonusDamage, 0, 0, 0;
```

Comment:
Current implementation for CNT leverage DEX + AGI for auto attack and STR + AGI for skill attack, this change will make AGI no longer relevant.

Reply:
and it's also have had critical damage scaling based on DEX that make it more necessary for normal attack (than hybrid with STR/AGI counterpart)